pkg/localstore/mode_put.go

```diff
@@ -274,6 +274,26 @@ func (db *DB) putUpload(batch *leveldb.Batch, binIDs map[uint8]uint64, item shed
 		return true, 0, nil
 	}
 
+	previous, err := db.postageIndexIndex.Get(item)
+	if err != nil {
+		if !errors.Is(err, leveldb.ErrNotFound) {
```

Comment:
reduce nesting by `if err != nil && !errors.Is(err,leveldb.ErrNotFound)`

Reply:
then the next if statement will need to recheck if `err == nil`. I do like the explicit nesting here :)

Reply:
fair enough :)